### PR TITLE
fixed eslint. added simple-git-hooks

### PR DESCRIPTION
**Pull request by ilajosmanov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,25 +35,29 @@
...
     "webpack": "^2.1.0-beta.28",
     "webpack-merge": "^1.1.2"
   },
   "dependencies": {
     "debounce": "^1.1.0"
+  },
+  "simple-git-hooks": {
+    "pre-commit": "npx lint-staged"
+  },
+  "lint-staged": {
```
Comment (dangreen) [inline package.json]: nano-staged

File: package.json [hunk 2]
```diff
@@ -34,25 +35,29 @@
...
+    "simple-git-hooks": "^2.7.0",
     "vue-loader": "^10.0.0",
     "vue-template-compiler": "^2.1.6",
     "webpack": "^2.1.0-beta.28",
     "webpack-merge": "^1.1.2"
   },
   "dependencies": {
     "debounce": "^1.1.0"
+  },
+  "simple-git-hooks": {
```
Comment (dangreen) [inline package.json:55]: Please move these configs to rc files

File: package.json [hunk 3]
```diff
@@ -53,11 +53,5 @@
     "debounce": "^1.1.0"
   },
   "simple-git-hooks": {
-    "pre-commit": "npx lint-staged"
-  },
-  "lint-staged": {
```
Comment (dangreen) [inline package.json]: > Add nano-staged and simple-git-hooks to lint files on commit and run tests on push

Why have you removed it?

**Review by dangreen — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (ilajosmanov) [reply package.json]: because in a next PR will push nano-staged

File: package.json (hunk 2 above)
Comment (ilajosmanov) [reply package.json:55]: hm, which pros?

**Pull request closed without merging**